lib/task-data/tasks/install-centos.js

```diff
@@ -8,8 +8,9 @@ module.exports = {
     implementsTask: 'Task.Base.Os.Install',
     options: {
         osType: 'linux', //readonly options, should avoid change it
-
         profile: 'install-centos.ipxe',
+        kickstart: 'centos-ks',
```

Comment:
I have some concern on the `kickstart`, `autoya``preseed` naming. 
It does looks very clearly, but make the `option` param `schema` different from OS. Also, we may end up with updating the `doc` to reflect this , `kickstart` is **required** only for CentOS, ESXi , `preseed` is **required** only for ubuntu ...

IMHO, the template file name itself is already clear to tell what the file type is. So any chance to unify the param key naming? For example, `configTemplate`, `templateUrl` ?